site/docs/v1/tech/installation-guide/monitor.adoc

```diff
@@ -6,27 +6,78 @@ description: Understand how your installation is operating by monitoring it.
 
 == Overview
 
-You can monitor FusionAuth in a number of ways.
+Once you've installed and integrated FusionAuth, you have to operate it. And for that you need insight into how the system is performing. There are a number of ways to do so.
 
-=== Log Files
+* <<Monitoring Summary Table>>
+* <<Log Files>>
+* <<Application Logging>>
+* <<Log File Formats>>
+* <<Application Events>>
+* <<Metrics>>
+** <<Prometheus Endpoint>>
+* <<Load Testing>>
+** <<Tips>>
+
+== Monitoring Summary Table
+
+Here's an overview table of monitoring options.
+
+.Monitoring Overview
```

Comment:
💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 

This is an awesome idea!  Thanks for writing it!

💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯